src/intersections.jl

```diff
@@ -203,7 +203,7 @@ function hasintersect(g1::Geometry{Dim,T}, g2::Geometry{Dim,T}) where {Dim,T}
 
   # initial direction
   c1, c2 = centroid(g1), centroid(g2)
-  d = c1 == c2 ? rand(Vec{Dim,T}) : c2 - c1
+  d = c1 ≈ c2 ? rand(Vec{Dim,T}) : c2 - c1
```

Comment:
I think we need to always use the verbose version with atol=atol(T) given that our atol is different than the default atol=0